ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -878,21 +874,21 @@ module.exports = Backbone.AssociatedModel.extend({
       return undefined
```

Comment:
✏️  consider putting () around `distance` =>  on line 871. 
what are those magic numbers on line 872.

Reply:
They're the valid bounds for ups coordinates. I can bring them into constants